test/nginx_system_test.sh

```diff
@@ -1315,7 +1315,7 @@ start_test Shutting down.
 fire_ab_load
 
 if $USE_VALGRIND; then
-    kill -s quit $VALGRIND_PID
+    kill -s TERM $VALGRIND_PID
```

Comment:
I think this switches us to a quick shutdown instead of a graceful one when testing a stressed shutdown under Valgrind. Is that intentional?